### PR TITLE
feat(experimental): `Button` add `icon` input for new icons approach, add Outline icons for it

### PR DESCRIPTION
**Pull request by waterplea**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]
left icons on api page is wrong

Снимок экрана 2023-11-10 в 11 59 37

File: projects/demo/src/modules/experimental/cell/examples/6/index.html [hunk 1]
```diff
@@ -48,8 +48,8 @@
         <button
             *ngFor="let item of items"
             appearance="opposite"
+            iconLeft="item.icon"
             tuiIconButton
```
Comment (splincode) [inline projects/demo/src/modules/experimental/cell/examples/6/index.html:52]: how is it work?

File: projects/demo/src/modules/experimental/cell/examples/6/index.html [hunk 2]
```diff
@@ -48,8 +48,8 @@
         <button
             *ngFor="let item of items"
             appearance="opposite"
+            iconLeft="item.icon"
```
Comment (splincode) [inline projects/demo/src/modules/experimental/cell/examples/6/index.html:51]: `[iconLeft]="item.icon"` ?

File: projects/demo/src/modules/experimental/icon/icon.style.less [hunk 3]
```diff
@@ -1,5 +0,0 @@
-@supports not (mask-mode: luminance) {
-    .luminance {
-        display: none;
```
Comment (nsbarsukov) [inline projects/demo/src/modules/experimental/icon/icon.style.less:3]: Delete this line too:

https://github.com/taiga-family/taiga-ui/blob/425be2617236e1b7cfd5107824e7e7e270515ece/projects/demo-playwright/tests/demo/is-flaky-examples.ts#L16

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**